src/resources/ResourceSnapshots/ResourceSnapshots.ts

```diff
@@ -193,9 +193,12 @@ export default class ResourceSnapshots extends Resource {
      * @description Shows the diff report for the target snapshot and dry-run report
      * @experimental
      */
-    diff(snapshotId: string, relativeReportId: string) {
+    diff(snapshotId: string, relativeReportId: string, numberOfLinesMax?: number) {
```

Comment:
Please document all the parameters.
I'm guessing there's a default for the new parameter in the backend, I'd like to see it documented using JSDOC `@default` (you can search for occurrences in the project if you need examples.)

Reply:
Its an optional value, you can call it with the param or without it

Reply:
It's ok on my side @louis-bompart and the API is update in environment dev now